Windows image no longer building

See

https://github.com/argoproj/argo-workflows/runs/5132490672?check_suite_focus=true

```
Step 13/26 : COPY --from=builder C:/ProgramData/chocolatey/lib/docker-cli/tools/docker.exe C:/app/docker.exe
[219](https://github.com/argoproj/argo-workflows/runs/5132490672?check_suite_focus=true#step:5:219)
COPY failed: stat C:\ProgramData\chocolatey\lib\docker-cli\tools\docker.exe: file does not exist
[220](https://github.com/argoproj/argo-workflows/runs/5132490672?check_suite_focus=true#step:5:220)
Error: Process completed with exit code 1.
```

--- a/Dockerfile.windows
+++ b/Dockerfile.windows
@@ -36,7 +36,7 @@ RUN mkdir C:\app && \
     curl -L -o C:\app\kubectl.exe "https://storage.googleapis.com/kubernetes-release/release/v%KUBECTL_VERSION%/bin/windows/amd64/kubectl.exe" && \
     curl -L -o C:\app\jq.exe "https://github.com/stedolan/jq/releases/download/jq-%JQ_VERSION%/jq-win64.exe"
 
-COPY --from=builder C:/ProgramData/chocolatey/lib/docker-cli/tools/docker.exe C:/app/docker.exe
+COPY --from=builder C:/ProgramData/chocolatey/lib/docker-cli/tools/docker/docker.exe C:/app/docker.exe
 COPY --from=builder C:/tools/git C:/app/git
 COPY --from=builder C:/ProgramData/chocolatey/lib/7zip.portable/tools/7z-extra/x64/7za.exe C:/app/7za.exe
 

